README.rst

```diff
@@ -7,90 +7,85 @@ What is the Atomic Developer Bundle (ADB)?
 ==========================================
 
 The Atomic Developer Bundle (ADB) is a prepackaged development
-environment filled with production-grade pre-configured tools that makes
-container developer's lives easier.  The ADB supports the development
+environment filled with production-grade, pre-configured tools, that makes
+container developers' lives easier.  The ADB supports the development
```

Comment:
Why this is not like `developer's` as you have used this pattern in "What is the Typical Usage Pattern?".